src/main.zig

```diff
@@ -18,6 +18,9 @@ const Save = @import("config/Save.zig");
 const migrator = @import("config/migrator.zig");
 const SharedError = @import("SharedError.zig");
 const utils = @import("tui/utils.zig");
+const c_import = @cImport({
+    @cInclude("unistd.h");
+});
```

Comment:
Please include this in `interop.zig` instead, and rename it `unistd`, like so:

```suggestion
const unistd = @cImport({
    @cInclude("unistd.h");
});
```